android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/PatientExtension.kt

```diff
@@ -38,6 +38,11 @@
   return calculateAge(birthDate, context)
 }
 
+fun Patient.extractPhone(context: Context): String {
+  if (!hasTelecom()) return "Contact not available"
```

Comment:
should we just retun "" to allow custom message via configs based on use context

Reply:
This is not necessary. Phone can be extracted via the rules configuration.

Reply:
resolved with
**fhirPath.extractValue(Patient, 'Patient.telecom[0].value')**
in no phonecontacts, its return ""